### PR TITLE
[ERE-990] Support code for registry specific quicklinks

### PR DESCRIPTION
**Pull request by alintreznai**

### CODE REVIEW

File: rdrf/registry/groups/models.py [hunk 1]
```diff
@@ -238,10 +243,26 @@ def set_password(self, raw_password):
         super().set_password(raw_password)
         self.force_password_change = False
 
+    def _load_quick_links(self):
+        valid_setting = hasattr(settings, 'QUICKLINKS_MODULE') and hasattr(settings, 'QUICKLINKS_CLASS')
```
Comment (ScriptSmith) [inline rdrf/registry/groups/models.py]: I think we should we add these + `EXTRA_WIDGETS` to settings.py. That way they're recorded somewhere.

Plus for quicklinks specifically, if we add defaults then we don't need to check if the value is set.

File: rdrf/registry/groups/models.py [hunk 2]
```diff
@@ -238,10 +243,26 @@ def set_password(self, raw_password):
...
+        if not valid_setting:
+            module = import_module('rdrf.forms.navigation.quick_links')
+            name = 'QuickLinks'
+        else:
+            module = import_module(settings.QUICKLINKS_MODULE)
```
Comment (ScriptSmith) [inline rdrf/registry/groups/models.py]: Django's `import_string` makes this simpler:

`import_string("mnd.forms.navigation.quick_links.MNDQuickLinks")`

File: rdrf/registry/groups/models.py (hunk 1 above)
Comment (alintreznai) [reply rdrf/registry/groups/models.py]: Updated.

File: rdrf/registry/groups/models.py (hunk 2 above)
Comment (alintreznai) [reply rdrf/registry/groups/models.py]: Updated.

**Review by ScriptSmith — APPROVED** [review]

**Pull request merged**